users/management/commands/unlock_and_generate_backup_code.py

```
@@ -21,39 +20,10 @@ def handle(self, *args, **options):
 
         disable_current_active_user = options.get("disable_current_active_user", True)
 
-        inactive_user = get_inactive_user(phone_number, inactive_user_id)
+        try:
+            inactive_user = get_inactive_user(phone_number, inactive_user_id)
+        except UnlockUserError as e:
+            raise CommandError(str(e))
```

Ruff (0.15.21)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)